src/domain/Forms/VerticalForm/VerticalForm.examples.md

```diff
@@ -317,3 +317,34 @@ initialState = { textInputValue: '' };
         </VerticalForm.RightAlignControls>
   </VerticalForm>
 ```
+
+Field with label suffix
+
+```jsx
+import { ButtonGroup, Button } from '@Domain/Buttons';
+import { TextInput } from '@Domain/Inputs';
+
+initialState = { textInputValue: '' };
+
+  <VerticalForm
+    onSubmit={() => alert(`Test input: ${state.textInputValue}`)}
+  >
+    <VerticalForm.Field
+      inputName='testInput'
+      label='This is a test input'
+      labelSuffix='only test condition'
```

Comment:
I'd like our examples to move more towards real examples - can you update the example to be what you're using it for if it makes sense in the example page out of context?